internal/precache/progress.go

```diff
@@ -61,7 +61,7 @@ func (p *Progress) Log() {
 
 func (p *Progress) Persist(filename string) {
 	data, _ := json.Marshal(p)
-	if err := os.WriteFile(filename, data, 0o700); err != nil {
+	if err := os.WriteFile(filename, data, 0o700); err != nil { //nolint:gosec
```

Comment:
@sakhoury suppressing it for now...but do you think we can make this `0o600`? I can make this change now if you would like.

Reply:
Why would it be executable? This is just persisting the progress file, right? Please make the change and run through an IBU

Reply:
updated to 0o600